### PR TITLE
[#2] Basic Structure

### PR DESCRIPTION
**Pull request by davecaos**

### CODE REVIEW

File: src/conferl.app.src [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{application, conferl,
+ [
+  {description,
+   "An awesome application called conferl"},
```
Comment (elbrujohalcon) [inline src/conferl.app.src]: Change it to something more meaningful

Comment (elbrujohalcon) [inline src/conferl.app.src]: …and match the readme.

File: src/conferl.app.src [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  {vsn, "0.1"},
```
Comment (elbrujohalcon) [inline src/conferl.app.src]: Start with `0.0.1`

File: src/conferl.erl [hunk 3]
```diff
@@ -0,0 +1,20 @@
+-module(conferl).
+-export([start/0, start/2, stop/0, stop/1]).
+
+%% application
+%% @doc Starts the application
+start() ->
+    {ok, _Started} = application:ensure_all_started(conferl).
```
Comment (elbrujohalcon) [inline src/conferl.erl]: We use [2 spaces for indentation](https://github.com/inaka/erlang_guidelines#spaces-over-tabs).

File: src/conferl_sup.erl [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+%% admin api
+start_link() ->
+  supervisor:start_link({local, ?MODULE}, ?MODULE, {}).
+
+%% behaviour callbacks
+init({}) ->
+  {ok,
+    {
+      {one_for_one, 5, 10}
+      ,[
```
Comment (gadgetci) [inline src/conferl_sup.erl]: According to **Elvis**:
> Missing space after "," on line 17

**Pull request merged**